### ISSUE TITLE
Investigate issue in Production with eef8dfdb-ed57-4ab0-add4-5e66d20a1d2f

### ISSUE THREAD

**Issue by diversemix** [elifesciences/elife-xpub#1656]
When I got in this morning this was stuck in the `MECA_EXPORT_PENDING` state.

I since downloaded from S3 and copied into the SFTP site and it looks fine now.

**Comment by diversemix** [comment]
Excerpt from logs:

```
{"level":"info","message":"Registered component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T09:36:02.798Z"}
{"level":"info","message":"Registered server component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T09:36:02.800Z"}
{"level":"info","message":"Registered component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T10:43:08.142Z"}
{"level":"info","message":"Registered server component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T10:43:08.145Z"}
{"level":"info","message":"Registered component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T10:48:37.213Z"}
{"level":"info","message":"Registered server component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T10:48:37.215Z"}
{"manuscriptId":"64090e41-6302-4477-ab37-16f22e5d2efb","level":"info","message":"Starting MECA export","timestamp":"2019-03-11T13:57:55.335Z"}
{"level":"info","message":"Manuscript 64090e41-6302-4477-ab37-16f22e5d2efb successfully exported","timestamp":"2019-03-11T13:57:56.477Z"}
{"manuscriptId":"97da753e-f1c5-4217-807c-6250cdb3a517","level":"info","message":"Starting MECA export","timestamp":"2019-03-11T14:32:39.691Z"}
{"level":"info","message":"Manuscript 97da753e-f1c5-4217-807c-6250cdb3a517 successfully exported","timestamp":"2019-03-11T14:32:42.926Z"}
{"manuscriptId":"2749e3a6-d315-4733-9bfd-c4611c1fb138","level":"info","message":"Starting MECA export","timestamp":"2019-03-11T14:47:48.820Z"}
{"level":"info","message":"Manuscript 2749e3a6-d315-4733-9bfd-c4611c1fb138 successfully exported","timestamp":"2019-03-11T14:47:51.847Z"}
{"manuscriptId":"84442150-f4cb-4a10-9a9d-38682cedfb6d","level":"info","message":"Starting MECA export","timestamp":"2019-03-11T14:51:03.317Z"}
{"level":"info","message":"Manuscript 84442150-f4cb-4a10-9a9d-38682cedfb6d successfully exported","timestamp":"2019-03-11T14:51:04.925Z"}
{"manuscriptId":"599680a5-fdff-4921-9b4d-eab45226a075","level":"info","message":"Starting MECA export","timestamp":"2019-03-11T16:03:22.943Z"}
{"level":"info","message":"Manuscript 599680a5-fdff-4921-9b4d-eab45226a075 successfully exported","timestamp":"2019-03-11T16:03:27.308Z"}
{"level":"info","message":"Registered component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T16:10:59.090Z"}
{"level":"info","message":"Registered server component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T16:10:59.092Z"}
{"level":"info","message":"Registered component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T17:13:58.916Z"}
{"level":"info","message":"Registered server component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T17:13:58.918Z"}
{"level":"info","message":"Registered component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T17:19:05.317Z"}
{"level":"info","message":"Registered server component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T17:19:05.319Z"}
{"manuscriptId":"c5f6a7cb-6a23-46ff-b5e3-b7058c3b0597","level":"info","message":"Starting MECA export","timestamp":"2019-03-11T19:31:46.228Z"}
{"level":"info","message":"Manuscript c5f6a7cb-6a23-46ff-b5e3-b7058c3b0597 successfully exported","timestamp":"2019-03-11T19:31:47.658Z"}
{"level":"info","message":"Registered component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T21:45:33.956Z"}
{"level":"info","message":"Registered server component @elifesciences/xpub-meca-export","timestamp":"2019-03-11T21:45:33.959Z"}
{"manuscriptId":"ad986df7-2b9e-443b-90df-d20c421c0578","level":"info","message":"Starting MECA export","timestamp":"2019-03-12T00:35:31.065Z"}
{"level":"info","message":"Manuscript ad986df7-2b9e-443b-90df-d20c421c0578 successfully exported","timestamp":"2019-03-12T00:35:33.081Z"}
{"manuscriptId":"eef8dfdb-ed57-4ab0-add4-5e66d20a1d2f","level":"info","message":"Starting MECA export","timestamp":"2019-03-12T05:26:24.481Z"}
{"level":"info","message":"Registered component @elifesciences/xpub-meca-export","timestamp":"2019-03-12T05:45:28.306Z"}
{"level":"info","message":"Registered server component @elifesciences/xpub-meca-export","timestamp":"2019-03-12T05:45:28.308Z"}

```

**Comment by diversemix** [comment]
Appears there are no errors in the log around this time

**Comment by diversemix** [comment]
Closing this off as I think this can be solved with #1650
